.gitignore

```diff
@@ -6,6 +6,7 @@ project.lock.json
 /staging/
 /Packages/
 *.nuget.props
+/test/tools/Modules/SelfSignedCertificate/
```

Comment:
Even though this module is for test, perhaps it's ok to just use `install-module -scope currentuser`?  Adding it to a repo path breaks our work towards separating tests from dependencies on the repo.

Reply:
I had that exact discussion with @JamesWTruher and argued that we shouldn't pollute the installed modules because PowerShell was build on a machine. But I will make the change

Reply:
we should *not* be polluting the user's space with anything that isn't explicitly testing our behavior for the current user. These tests are very likely to be run on a dev box and we should *not* be adding stuff to the current user's scope. Further, this is a tool *only* for our tests. If the user happens to have this already, it still shouldn't use it, we should be using the module that we know we should be using in the location that we control without molesting their environment.